k8s/demo/common/ccd/latest-ccd-chart.yaml

```diff
@@ -33,6 +33,16 @@ spec:
       # can redirectUri be removed?
       redirectUri: https://ccd-demo-case-management-web.demo.platform.hmcts.net/oauth2redirect
       s2sUrl: http://ccd-demo-s2s
+
+    # Enable support services tags  
+    ccd:
+      printService:
+        enabled: true
+      activityApi:
+        enabled: true
+      paymentApi:
```

Comment:
paymentApi is not needed, it's already enabled on the product chart.
Also, this whole block should be only on the product chart. Please add a todo to remove it in the future and enable `activityApi` and `printService` permanently on the product chart

Reply:
fixed comments